pkgs/by-name/ou/outguess/package.nix

```
@@ -0,0 +1,36 @@
+{ fetchFromGitHub
+, stdenv
+, lib
+, autoconf
+, automake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "outguess";
+  version = "0.4";
+  src = fetchFromGitHub {
+    owner = "resurrecting-open-source-projects";
+    repo = "outguess";
+    rev = version;
+    hash = "sha256-yv01jquPTnVk9fd1tqAt1Lxis+ZHZqdG3NiTFxfoXAE=";
+  };
+
+  nativeBuildInputs = [ autoconf automake ];
```

```suggestion
  nativeBuildInputs = [ autoReconfHook ];
```

Try remove buildPhase, should work

---

It worked, just needed to add `configureFlags = [ "--with-generic-jconfig" ];`